### PR TITLE
Various enhancements

### PR DESCRIPTION
**Pull request by astrolox**
- Up to date fleshed out README
- Additional PHP modules (some requested by users, others just because it seems reasonable)
- Web server syntax checking on build
- Better consistency with the other PHP images
- Fully tested locally (using make)
- General tidying up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+UID=999
+PORT=80
+WEB_ROOT="/var/www/"
+
+docker run -u ${UID}:0 -p ${PORT}:8080 -v ${WEB_ROOT}:/var/www/ 1and1internet/ubuntu-16-nginx-php-5.6
+```
+
+## Building and testing
+
+A simple Makefile is included for your convience. It assumes a linux environment with a docker socket avialable at `/var/run/docker.sock`
```
Comment (mintopia) [inline README.md]: Typo - 'available'

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+Please consult [the official Ubuntu site](https://www.ubuntu.com/info/release-end-of-life) for information on when this version of Ubuntu becomes end of life.
+
+Please consult [the official PHP site](http://php.net/supported-versions.php) to know when this version of PHP becomes end of life. It is recommended that you move to a newer version of PHP once this version passed out of active support.
+
+***PHP 5.6 stopped receiving active support on 19 Jan 2017.***
+
+## Usage
+
+Please note this image is explictly intended to be run as a non-privildged user. Ensure you specify a user id (UID) other than zero when you run it. Running as root will not function.
```
Comment (mintopia) [inline README.md]: Typo - 'privileged'

**Comment by mintopia** [comment]
lgtm

**Comment by hrobertson** [comment]
lgtm

**Comment by hnorobertson** [comment]
lgtm

**Pull request merged**